regression/storage.py

```diff
@@ -52,15 +53,16 @@ def setUp(self):
         self.case_buckets_to_delete = []
 
     def tearDown(self):
-        for bucket in self.case_buckets_to_delete:
-            bucket.delete()
+        with Batch(CONNECTION) as batch:
```

Comment:
+1 on this, though it could have waited for another PR.

We could also look into batch support for `Bucket.delete(force=True)`.